recipe/meta.yaml

```diff
@@ -13,15 +13,17 @@ source:
   md5: 8987b9a3e3cd79218a0a423b21c8e4de
 
 build:
-  number: 200
+  number: 201
   # We lack openblas on Windows, and therefore can't build scipy there either currently.
   skip: true  # [win]
   features:
     - blas_{{ variant }}
 
 requirements:
   build:
+    - toolchain
     - gcc
+    - cloog 0.18.0 10
```

Comment:
Thanks @jakirkham. Maybe add a comment in the recipe explaining the purpose of ``cloog``?